internal/api/handlers/schedulers_handler.go

```diff
@@ -102,6 +102,21 @@ func (h *SchedulersHandler) AddRooms(ctx context.Context, request *api.AddRoomsR
 	}, nil
 }
 
+func (h *SchedulersHandler) RemoveRooms(ctx context.Context, request *api.RemoveRoomsRequest) (*api.RemoveRoomsResponse, error) {
+
+	operation, err := h.schedulerManager.RemoveRooms(ctx, request.GetSchedulerName(), request.GetAmount())
+	if errors.Is(err, portsErrors.ErrNotFound) {
```

Comment:
Note (not really actionable): I think we are starting to "leak" those internal errors (port errors) into the layers we shouldn't (like the API handlers here). We might need to discuss introducing a `errors` package for the services or develop a different solution.

Reply:
@gabrielcorado Either way we would have our API layer dealing with internal errors from the Core layer since our services are core domain logic, I don't see a problem with that. But I get your point about having a "translation" of internal errors to more "semantic" ones, defining and isolating the possible errors that our `services` could return, I like this idea.

Reply:
You're right, and the API still relies on the core errors. However, we could rely on errors from the closest layer (in this case, the services) instead of some deep errors (like the ports one).

Anyway, WDYT of taking it as technical debt for now, and we can tackle it later when we have more samples of this use case?

Reply:
I Agree! :)